src/ttsnake.c

```diff
@@ -620,9 +627,9 @@ void playgame (scene_t* scene, char *data_dir)
       clear ();                               /* Clear screen. */
       refresh ();			      /* Refresh screen. */
 
-      if(!on_settings) {
+      if(!on_settings && !pause_game) {
         advance (scene);		               /* Advance game.*/
-      } else {
+      } else if (!on_settings) {
```

Comment:
Settings aren't working properly for me. I think this should be `if(on_settings)`. I will fix it if it's just this